src/ContentImporter.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\openy;
+
+use Drupal\migrate\MigrateMessage;
+use Drupal\migrate\Plugin\Migration;
+use Drupal\migrate_tools\MigrateExecutable;
+
+/**
+ * Class ContentImporter.
+ *
+ * @package Drupal\openy
+ */
+class ContentImporter implements ContentImporterInterface {
+
+  /**
+   * Migration manager.
+   *
+   * @var \Drupal\migrate_plus\Plugin\MigrationConfigEntityPluginManager
+   */
+  protected $migrationManager;
+
+  /**
+   * Map of content items and migrations.
+   *
+   * @var array
+   */
+  protected $map;
+
+  /**
+   * ContentImporter constructor.
+   */
+  public function __construct() {
+    $this->map = $this->getMap();
+
+    // Service container are not available on the first steps of profile installation.
+    $this->migrationManager = \Drupal::service('plugin.manager.config_entity_migration');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMap() {
```

Comment:
We should also trigger migration for footer blocks https://github.com/ymcatwincities/openy/tree/8.x-1.x/modules/openy_features/openy_demo_content/modules/openy_demo_bfooter, but there's dependency on block configuration. So, could be tricky to resolve dependencies.